test/schema_test.mjs

```diff
@@ -398,3 +399,19 @@ test("should be a valid arweave message", async (t) => {
   const valid = check(message);
   t.true(valid);
 });
+
+test("arweave gateway should be a https url", async (t) => {
+  const check = ajv.compile(arweave);
+  const message = {
+    options: {
+      uri: "ar://ltmVC0dpe7_KxFHj0-S7mdvXSfmcJOec4_OfjwSzLRk/1",
+      gateway: "arweave.net/"
```

Comment:
mhh very interesting that format: uri allows this, as it should mandate `https://`. I'll take this PR over and do some experiments